### ISSUE TITLE
BodyHeatmap missing aria-labels on right shoulder and right quadriceps regions

### ISSUE THREAD

**Issue by aschung212** [aschung212/Lift#78]
> Migrated from Linear issue MAS-333 on 2026-04-03

**Project:** Lift
